test/integration/008_schema_tests_test/test_schema_v2_tests.py

```diff
@@ -414,8 +412,8 @@ def test_postgres_collision_test_names_get_hash(self):
 
         # both tests have the same unique id except for the hash
         expected_unique_ids = [
-            'test.test.not_null_base_extension_id.2dbb9627b6',
-            'test.test.not_null_base_extension_id.d70fc39f40'
-            ]
+            'test.test.not_null_base_extension_id.4a9d96018d',
+            'test.test.not_null_base_extension_id.60bbea9027'
```

Comment:
Do you know why the `unique_id`s have changed? We shouldn't be including any configs in the hash, right?

Reply:
The model kwarg changed with the `where` config. The hash includes kwargs.